README.md

```diff
@@ -6,7 +6,11 @@ Organizational repository for the meetups initiave in OSDC
 * [v1shvesh](https://github.com/v1shvesh) - **S Vishvesh** &lt;`<maxim260598@gmail.com>`&gt;
 * [Sudeepam97](https://github.com/Sudeepam97) - **P Sudeepam** &lt;`<sudeepam.pandey@gmail.com>`&gt;
 
-## Linux Meetup 0
+## Meetups
 
-Linux Introductory talk given by [crapsystem](https://github.com/crapsystem) (Himanshu Nailwal) in the first hour.
-A CTF based linux introduction game using simple shell commands was hosted in the final hour.
+List of all the meetups of the OSDC community.
+
+| S.No. | Date | Meetup Agenda | Details | 
+| :---: | :--: | ------ | ---------- |
+|   1   | 30-Aug-2018 | [Wanna Play Linux](./linux-meetup-0) | Linux Introductory talk given by [crapsystem](https://github.com/crapsystem) (Himanshu Nailwal) in the first hour.A CTF based linux introduction game using simple shell commands was hosted in the final hour.|
```

Comment:
@sidntrivedi012 Let's use `DD-MM-YY`?

Reply:
Current format looks good. Can you increase its width so that the whole date is in one line ?